src/pages/GE_TokenizeCard.page

```diff
@@ -43,6 +43,9 @@
             .error-wrapper.fieldset.cvc {
                 max-width: fit-content;
             }
+            .ACHInput__ButtonContainer-sc-6riaoe-0 {
```

Comment:
Is there any chance this class name could change?  The code at the end (sc-6riaoe-0) kind of looks like it could be dynamically generated.  Is there any documentation or anything that would allow us to be confident that the element that is being set to display:none will always have this class?

Reply:
Good catch @kenlewis-sfdo. It's been updated.

Reply:
Awesome - thanks!